test/framework/src/main/java/org/elasticsearch/AnalysisFactoryTestCase.java

```diff
@@ -228,6 +228,10 @@
         .put("fingerprint",               Void.class)
         // for tee-sinks
         .put("daterecognizer",            Void.class)
+        .put("minhash",                   Void.class)
```

Comment:
for the record 6.2 is the first release that the scandinavian normalization filters support multi-term analysis https://issues.apache.org/jira/browse/LUCENE-7353